egs/ami/s5/local/nnet3/run_lstm.sh

```diff
@@ -10,28 +10,27 @@ set -e
 stage=0
 train_stage=-10
 has_fisher=true
-mic=ihm
-use_sat_alignments=true
+mic=sdm1
```

Comment:
this is a common script used for sdm*, ihm and mdm8. Please don't change the default mic. The default recipe for ihm uses sat alignments so don't change use_sat_alignments.